zeek

```diff
@@ -140,10 +142,10 @@ main() {
 		while IFS=  read -r -d $'\0' CONFIG; do
 			docker_cmd+=("--mount" "source=$CONFIG,destination=/usr/local/zeek/${CONFIG#"$HOST_ZEEK"},type=bind")
 		done < <(find "$HOST_ZEEK/etc/" -type f -print0 2>/dev/null)
-		# mount all zeek scripts
+		# mount all zeek scripts, except local.zeek which will be auto-generated instead
 		while IFS=  read -r -d $'\0' SCRIPT; do
 			docker_cmd+=("--mount" "source=$SCRIPT,destination=/usr/local/zeek/${SCRIPT#"$HOST_ZEEK"},type=bind")
-		done < <(find "$HOST_ZEEK/share/" -type f -print0 2>/dev/null)
+		done < <(find "$HOST_ZEEK/share/" -type f ! -name local.zeek -print0 2>/dev/null)
```

Comment:
The old local.zeek file will still exist upon an upgrade. Perhaps we should add a bit of logic to this script to archive the old version? (Rename it with .bak and maybe add a header comment?)